common/app/views/fragments/containers/facia_cards/container.scala.html

```diff
@@ -36,6 +37,20 @@
             }
         }
 
+        case _ if frontId.get == AudioFlagship.tagName => {
+            <section id="@componentName"
```

Comment:
This can be moved into `audioFlagshipContainer.scala.html`, which can then be called directly from `indexBody.scala.html`